drivers.schema.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Sorry for adding a comment again, but I found this is not the latest version. The official document says to use: `https://json-schema.org/draft/2020-12/schema`
https://json-schema.org/learn/getting-started-step-by-step#create-a-schema-definition